server/szurubooru/func/posts.py

```diff
@@ -421,6 +426,15 @@ def create_post(
     return post, new_tags
 
 
+def update_post_file_last_modified_time(
+    post: model.Post, timestamp: int
+) -> None:
+    assert post
+
+    # Timestamp is in ms, so it must be divided by 1000 otherwise out of range
+    post.file_last_modified_time = datetime.fromtimestamp(timestamp / 1000)
```

Comment:
rename the variable `timestamp` to `timestamp_in_ms`, code becomes self-documenting and then you can remove the comment

Reply:
I'm working on that file, I will do that. There's one thing preventing me from finishing the commit, though, how do you get the path to an image in `image.py`?

Reply:
Believe the helper function for that is this: https://github.com/rr-/szurubooru/blob/0dd427755b601da8dadf9af1a34ecad03f8446f9/server/szurubooru/func/posts.py#L127-L134

Reply:
Ah, thank you!